src/raftstore/store/fsm/metrics.rs

```diff
@@ -11,6 +11,12 @@ lazy_static! {
         exponential_buckets(1.0, 2.0, 20).unwrap()
     )
     .unwrap();
+    pub static ref WORKER_POLL_HANDLE_BATCH_COUNTER: CounterVec = register_counter_vec!(
+        "tikv_raftstore_poll_handle",
+        "The count of batch handle by every thread worker poll",
```

Comment:
This description doesn't read great in english. Maybe it's clear to a more experienced user, but I don't know what it is saying.

I might _guess_ that it could be stated as "The total count of task batches handled by each thread of the raft storage worker pool". Now, whether those words actually make sense technically, and whether that is what is being counted, I have no idea, but the sentence reads clearer to me.

Reply:
thx